src/groupByKey.ts

```diff
@@ -1,7 +1,21 @@
 interface Item {
-  [key: string]: any;
+  [key: string] : any,
 }
 
-export function groupByKey(items, field) {
-  // write code here;
+interface GropedItems {
+  [key: string] : Item[],
+}
+
+export function groupByKey(items: Item[], field: keyof Item):Item {
```

Comment:
```suggestion
export function groupByKey(items: Item[], field: keyof Item): GroupedItems {
```
The function should return the object of proper type